### DISCUSSION TITLE
A second account

### DISCUSSION THREAD

**Discussion by olek-b-v** [codewars/codewars.com#3215] [Q&A] [answered]
As a part of a job interview I have been asked to create a new account and complete a number of codewars katas. The criteria for completing the given task was "The account you’ve created must contain only listed below kata".
Some of the katas I have already completed in my main account. Am I allowed to use my own solutions (all original code with no crossings with any other users, I have never skipped any katas in neither accounts)  from the main account in the interview account? Or will it count as a violation of rules? I won't even need this account in 2 weeks (after passing an interview), but right now I have no choice.

**Comment by DonaldKellett** [comment]
You should recommend Qualified to your potential employer which is much
better suited for the interview use case: https://www.qualified.io/

Copying your solutions from your "main" account to your "interview"
account, strictly speaking, is a ToS violation and can result in one or
both accounts being restricted.

**Reply by olek-b-v** [reply]
Qualified does look interesting, but I have no influence on the interviewing process. 
This seems like an excessively strict policy. It is my own original code, what would be the reason for penalizing me? I take no part in any ranking systems, so it wouldn't impact anyone else. And I have no plans to further use my 'interview' account.

**Reply by hobovsky** [reply, marked as answer]
Main reason for penalizing you would be the fact that moderators do not know that both account are yours, and the "interview" account would look from our perspective like someone copying solutions from your main account.

You can help by giving the interview account some similar user name, for example your main account would be "Olek", and the interview account would be "Olek-Interview" - this could give mods a hint. You let us know here too, and this also can be helpful. Maybe a small note in the "Skills" or "Name" fields that both accounts are connected.

Ideally, you would use your main account for the interview. If you can't, you can create another one, and make it obvious that it is also yours. You risk that some moderator unaware of this conversation will take some action on it, but maybe we could handle this internally until your interview is done. Then you would delete the interview account.

**Reply by olek-b-v** [reply]
Thank you very much! I will do as you suggested.
